### PR TITLE
fix: added eslint rule, refactor codebase with specified requirements

### PR DESCRIPTION
**Pull request by 0tuedon**
- [x] Remove unuseful comments
- [x] remove eslint  ignore rules from entire file
- [x] no  tag use <Image> instead
- [x] remove filtered by transcripts

Pending on the next PR 
using promise.all for user reviews in current jobs table

### CODE REVIEW

**Comment by Extheoisah** [comment]
The images in the Sats column are not centered.
image

**Comment by 0tuedon** [comment]
> The images in the Sats column are not centered. image

This current push fixes this, i added an ml on larger screens to fixe this

File: src/components/tables/Pagination.tsx [hunk 1]
```diff
@@ -1,19 +1,32 @@
...
 type IPagination = {
   pages: number;
   currentPage: number;
   setCurrentPage: Dispatch<SetStateAction<number>>;
 };
 const Pagination = ({ pages, currentPage, setCurrentPage }: IPagination) => {
+  const router = useRouter();
+  const handleNextPage = (page: number) => {
+    router.replace({
+      query: { ...router.query, pageNum: page },
```
Comment (Extheoisah) [inline src/components/tables/Pagination.tsx:14]: let's just call this page
```suggestion
      query: { ...router.query, page },
```

File: src/components/tables/Pagination.tsx [hunk 2]
```diff
@@ -1,19 +1,32 @@
...
+    });
+    setCurrentPage(page);
+  };
+  useLayoutEffect(() => {
+    if (router.query?.pageNum) {
```
Comment (Extheoisah) [inline src/components/tables/Pagination.tsx:19]: same here
```suggestion
    if (router.query?.page) {
```

File: src/components/tables/Pagination.tsx (hunk 1 above)
Comment (0tuedon) [reply src/components/tables/Pagination.tsx:14]: okay

**Comment by 0tuedon** [comment]
https://github.com/bitcointranscripts/transcription-review-front-end/assets/90271995/25e529ad-7587-4c29-8e80-0480377f77ce

Changed pageNum to page

**Review by Extheoisah — APPROVED** [review]

**Pull request merged**